pages/learn.vue

```diff
@@ -158,14 +173,37 @@ export default class extends QiskitPage {
     this.$store.commit('setLearnLevel', level)
   }
 
-  get isAMinuteForBeginner (): boolean {
-    return (this as any).timeScale === TIME_SCALES.minute &&
-      (this as any).learnLevel === LEARN_LEVELS.beginner
+  showExplanation (level: 'beginner'|'advanced') {
+    this.$store.commit('setTimeScale', TIME_SCALES.minute)
+    this.$store.commit('setLearnLevel', level)
+  }
+
+  get showingEverything (): boolean {
+    const { timeScale, learnLevel } = (this as any)
+    return timeScale === TIME_SCALES.any && learnLevel === LEARN_LEVELS.all
   }
 
-  get isAMinuteForAdvanced (): boolean {
-    return (this as any).timeScale === TIME_SCALES.minute &&
-      (this as any).learnLevel === LEARN_LEVELS.advanced
+  get showingOneMinuteForBeginner (): boolean {
```

Comment:
What about create `showingOneMinuteFor(level)` to replace `showingOneMinuteForBeginner` and `showingOneMinuteForAdvance`? You can access `this.showingLevel(LEARN_LEVELS.beginner)` with `this.showingLevel(LEARN_LEVELS[level])`